cxx4opencl/intro.txt

```diff
@@ -7,7 +7,12 @@
 
 This language is built on top of OpenCL C v2.0 and {cpp}17 enabling
 most of regular {cpp} features in OpenCL kernel code. Most functionality
-from OpenCL C and {cpp} is inherited.
+from OpenCL C and {cpp} is inherited. Since both OpenCL C and {cpp} are
```

Comment:
"OpenCL C and C++" could be interpreted as "OpenCL C and OpenCL C++", even though you do not intend this.

I'd suggest to either swap the order everywhere ("C++ and OpenCL") or explicitly mention C++17 to avoid the ambiguity.

Reply:
Good thinking. I have swapped them.

Reply:
I have applied it manually now. I think I have forgotten about this somehow.